docs/platforms/python/configuration/filtering/index.mdx

```diff
@@ -16,9 +16,40 @@ Configure your SDK to filter error events by using the <PlatformIdentifier name=
 
 ### Using <PlatformIdentifier name="before-send" />
 
-All Sentry SDKs support the <PlatformIdentifier name="before-send" /> callback method. Because it's called immediately before the event is sent to the server, this is your last chance to decide not to send data or to edit it. <PlatformIdentifier name="before-send" /> receives the event object as a parameter, which you can use to either modify the event’s data or drop it completely by returning `null`, based on custom logic and the data available on the event.
+The `before_send` callback allows you to modify the event payload before the SDK sends the event to Sentry. You can also stop the event from being sent by returning `None`.
 
-<PlatformContent includePath="configuration/before-send/" />
+The callback receives two arguments: the event payload and a [hint](#event-hints). The callback can return either the event payload to send to Sentry or `None` if the event should be dropped.
+
+Since the SDK sends to Sentry the exact payload that the callback returns, the callback can perform arbitrary modifications to the event payload, including adding data to the event and modifying or deleting existing event fields. It is even possible to use `before_send` to construct a completely different event payload from scratch.
```

Comment:
```suggestion
Since the SDK sends to Sentry the exact payload that the callback returns, the callback can perform arbitrary modifications to the event payload, including adding data to the event and modifying or deleting existing event fields.
```

I would not tell users this, so they do not have bad ideas :-)